tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1098,6 +1098,12 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange:
     conditions:
       - "asic_type in ['mellanox']"
 
+qos/test_qos_sai.py::TestQosSai::testQosSaiFullMeshTrafficSanity:
+  skip:
+    reason: "Unsupported testbed type."
+    conditions:
+      - "topo_name not in ['ptf64'] and asic_type in ['cisco-8000']"
```

Comment:
From your code here, the case should be skipped if asic type is cisco-8000, how you get the test result you posted?

Reply:
@kevinskwang the case will be skipped only if it's not a ptf64 topology and asic-type is cisco-8000.  The test result is posted from a testbed having ptf64 topology and asic_type is cisco-8000

Reply:
yes it can be used by other platforms as long as it is from Cisco.

Reply:
No, I mean other platform, like broadcom. mellanox

Reply:
ok. I got it now. Yes, it's written currently for Cisco platform only. 

I have fixed the condition to skip for other platforms too.